Gemfile.lock

```diff
@@ -74,10 +74,10 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: a6815eb11dde49c728182e9fd2370410a37ad005
+  revision: 829e126f931d51341e5377b98e9ffdee73d3f2e9
   branch: master
   specs:
-    vets_json_schema (24.13.3)
+    vets_json_schema (24.14.0)
```

Comment:
matches what's in master: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/package.json#L3